docs/report.md

```diff
@@ -26,13 +26,48 @@ Our domain model consists of two data entities, which depict the attributes of a
 
 ![Illustration of the _Chirp!_ data model as UML class diagram.\lable{domainModelImage}](https://raw.githubusercontent.com/ITU-BDSA23-GROUP12/Chirp/main/docs/images/DomainModel.drawio1.png)
 
-
 ## Architecture — In the small
 
 ## Architecture of deployed application
 
 ## User activities
-
+The gist of our application lies in two main activities; writing cheeps and reading cheeps. A **cheep** is a short message of 1-160 characters, and is not messaged to a particular person. Each cheep is publicly available for everyone to see, and there is not functionality to direct the attention of particular users onto your cheep. Lastly, the user has an option to follow other users, to get easier access to cheeps written by them. \ref{Authenticated Public Timeline}
+The use of our application is defined by a range of tabs, a **cheep box** and two different kinds of users - **authenticated** users and **unauthenticated** users. That is, users that are logged in and users that are not. The two kinds of users differ in the elements that they have access to on the site. 
+Below is a screenshot of the main page and then a short description of the different elements.
```

Comment:
set din reference til dit billlede her: 
"Below In figure \ref{Authenticated Public Timeline}..."